src/components/applicant/ApplicantInfoContent.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from 'react';
+
+import './ApplicantInfo.scss';
+import ApplicantInfoHeader from './ApplicantInfoHeader';
+import ApplicantViewerContentComponent from './ApplicantViewerContentComponent';
+
+interface IApplicantInfoContentProps {
+    email: string;
+    year: number;
+    major: string;
+    exposure: string;
+    resume: string;
+    github: string;
+    website: string;
+    interest: string;
+    mainPlatform: string;
+    allPlatforms: string;
+    project: string;
+    projectLink: string;
+}
+
+const ApplicantInfoContent: FunctionComponent<IApplicantInfoContentProps> = (props: IApplicantInfoContentProps) => {
+    return (
+        <React.Fragment>
+            <div className="container">
+                <ApplicantInfoHeader
+                    email={props.email}
+                    year={props.year}
+                    major={props.major}
+                    exposure={props.exposure}
+                    resume={props.resume}
+                    github={props.github}
+                    website={props.website}
+                />
+                <ApplicantViewerContentComponent
+                    interest={props.interest}
+                    mainPlatform={props.mainPlatform}
+                    allPlatforms={props.allPlatforms}
+                    project={props.project}
+                    projectLink={props.projectLink}
```

Comment:
I think a more dynamic way to do this is to have lets say like 5 ApplicantInfoContent containers. These ApplicantInfoContent containers will have a heading, and a description to them rather than having to hard code in the descriptions.

So instead of there being a prop that contains things like props.interest or props.mainPlatform,

Lets just have an array of objects that contain a heading, and description, so we can reuise the same component several times over and just copy in props.header and props.description.